src/document-converter.ts

```diff
@@ -1098,6 +1113,15 @@ export class DocumentConverter {
           'shadycss/custom-style-interface.html',
           'shadycss/entrypoints/custom-style-interface.js');
     }
+    if (this.originalPackageName === 'polymer') {
+      if (jsUrl === './polymer.html') {
+        jsUrl = jsUrl.replace('polymer.html', 'polymer-legacy.html');
+      }
+      if (jsUrl.endsWith('polymer/polymer.html')) {
+        jsUrl = jsUrl.replace(
```

Comment:
The replace is a bit loose compared to the endsWith check. Add an EOL anchor: `jsUrl = jsUrl.replace(/\.html$/, '-legacy.html')`